src/handler.py

```diff
@@ -1,123 +1,142 @@
+import json
 import os
 import boto3
 import datetime
 import logging
 
+from src.rds import RDS
+
+log_level = os.getenv('LOG_LEVEL', logging.ERROR)
 logger = logging.getLogger()
-logger.setLevel(logging.DEBUG)
+logger.setLevel(log_level)
+
 two_days_ago = datetime.datetime.now() - datetime.timedelta(2)
 
+"""
+This is supposed to be all lambda functions invoked by the state machine.
+"""
+LAMBDA_FUNCTIONS = [
+    'aqts-capture-raw-load-TEST-iowCapture',
+    'aqts-ts-type-router-TEST-determineRoute',
+    'aqts-capture-ts-description-TEST-processTsDescription',
+    'aqts-capture-ts-corrected-TEST-preProcess',
+    'aqts-capture-ts-field-visit-TEST-preProcess',
+    'aqts-capture-field-visit-metadata-TEST-preProcess',
+    'aqts-capture-field-visit-transform-TEST-transform',
+    'aqts-capture-discrete-loader-TEST-loadDiscrete',
+    'aqts-capture-dvstat-transform-TEST-transform',
+    'aqts-capture-ts-loader-TEST-loadTimeSeries',
+    'aqts-capture-error-handler-TEST-aqtsErrorHandler'
+]
+
 month = str(two_days_ago.month)
 if len(month) == 1:
     month = f"0{month}"
 day = str(two_days_ago.day)
 if len(day) == 1:
     day = f"0{day}"
 
-# Default snapshot identifier, may be overridden in restoreDbCluster
+# Default snapshot identifier, may be overridden by passing a custom
+# snapshot identifier in the step function event
 SNAPSHOT_IDENTIFIER = f"rds:nwcapture-prod-external-{two_days_ago.year}-{month}-{day}-10-08"
 DB_INSTANCE_IDENTIFIER = 'nwcapture-load-instance1'
 DB_INSTANCE_CLASS = 'db.r5.8xlarge'
 ENGINE = 'aurora-postgresql'
 DEST_BUCKET = 'iow-retriever-capture-load'
 SRC_BUCKET = 'iow-retriever-capture-reference'
 DB_CLUSTER_IDENTIFIER = 'nwcapture-load'
+NWCAPTURE_TEST = 'NWCAPTURE-DB-TEST'
+NWCAPTURE_LOAD = 'NWCAPTURE-DB-LOAD'
+
+TEST_LAMBDA_TRIGGERS = [
+    'aqts-capture-trigger-TEST-aqtsCaptureTrigger', 'aqts-capture-trigger-tmp-TEST-aqtsCaptureTrigger']
```

Comment:
I've successfully removed the `aqts-capture-trigger-tmp-TESt-aqtsCaptureTrigger` function. It can be removed from this list.